mcl_3dl:
  map_frame: "map"
  publish_tf: true
  output_pcd: false
  map_chunk: 20.0
  tf_tolerance: 0.05
  skip_measure: 1
  accum_cloud: 2
  lpf_step: 0.9
  acc_lpf_step: 128.0
  acc_var: 0.785398
  jump_dist: 1.0
  jump_ang: 1.57
  fix_dist: 0.5
  fix_ang: 0.3
  bias_var_dist: 2.0
  bias_var_ang: 1.57
  match_output_dist: 0.8
  unmatch_output_dist: 0.5
  match_output_interval_interval: 0.2
  dist_weight_x: 1.0
  dist_weight_y: 1.0
  dist_weight_z: 5.0
  num_particles: 250 ##256
  resample_var_x: 0.05
  resample_var_y: 0.05
  resample_var_z: 0.01
  resample_var_roll: 0.05
  resample_var_pitch: 0.05
  resample_var_yaw: 0.1
  expansion_var_x: 0.2
  expansion_var_y: 0.2
  expansion_var_z: 0.2
  expansion_var_roll: 0.05
  expansion_var_pitch: 0.05
  expansion_var_yaw: 0.03
  match_ratio_thresh: 0.01 #0.08
  odom_err_lin_lin: 0.05
  odom_err_lin_ang: 0.02 #0.05
  odom_err_ang_lin: 0.03
  odom_err_ang_ang: 0.03
  odom_err_integ_lin_tc: 10.0
  odom_err_integ_lin_sigma: 100.0
  odom_err_integ_ang_tc: 10.0
  odom_err_integ_lin_sigma: 100.0
  init_var_x: 2.0
  init_var_y: 2.0
  init_var_z: 0.01
  init_var_roll: 0.01
  init_var_pitch: 0.01
  init_var_yaw: 0.5
  init_x: 0
  init_y: 0
  init_z: 0
  init_roll: 0
  init_pitch: 0
  init_yaw: 0

  likelihood:
    match_dist_min: 0.2
    match_dist_flat: 0.05
    clip_near: 2.0
    clip_far: 100.0 #30
    num_points: 100
    num_points_global: 15

obj_to_pointcloud:
  frame_id: "map"
  points_per_meter_sq: 1200.0
  downsample_grid: 0.1
pcd_to_pointcloud:
  frame_id: "map"
  latch: true
